dags/common/utils.py

```diff
@@ -67,3 +67,14 @@ def extract_text(article, path, field_name, dois):
     except AttributeError:
         logger.error(f"{field_name} is not found in XML", dois=dois)
         return
+
+
+def remove_excluded_folders(filenames, excluded_folders):
```

Comment:
I don't think we need this function, in the walktree we can just pass the directories to ignore and that's it

Reply:
but this it is already in SFTP, should we change the SFTP?